s2i/run

```diff
@@ -2,11 +2,11 @@
 
 set -e
 
-# We will turn off daemonizing for the nginx process so that the container
-# doesn't exit after the process runs.
 echo -e "Running as user $(id)"
-echo "starting nginx server with daemon off"
-exec /usr/sbin/nginx -g "daemon off;"
+
+echo "Maybe you don't really want to do this and use a chained build instead?"
```

Comment:
I like the casual tone, but maybe we can point to a doc url here and note that running the container really only makes sense for test and dev?

Reply:
Yea,  i'll do that